### ISSUE TITLE
[1.12.2] Crashing with MixinBootstrap

### ISSUE THREAD

**Issue by Aaronek10** [OreCruncher/DynamicSurroundings#703]
**Mod Version:** DynamicSurroundings-1.12.2-3.6.1.0.jar

**Forge Version:** 14.23.5.2814

**Link to client log:** https://pastebin.com/RfWyv0fV

**Link to crash log:** https://pastebin.com/AfRMtLx3

**Description:**
After installing new version with dependencies like OreLib or MixinBootsrap game crashes. i read some closes issues and the solution should be installing MixinBootstrap. But that sadly didn't fixed anything.

* Did you just add the mod to a pack?
**Yes.**
* Did it just start crashing, or has it always crashed?
**Always crashed when i tried to launch game.**
* Did you update any other mods in your pack and it started crashing?  This includes Forge.
**Nope, only DynamicSurroundings.**

* If updating multiple mods did you update one by one or all at once?  To help narrow the problem down adding one by one is helpful.
**Only one by one.**

* Is the modpack you are using available online somewhere?  If the crash is due to mod interaction having the pack available makes things much easier.  It is much harder to work these types of issues with a private kitchen sink pack containing 250 mods.
**Nope... mod count didn't reached 60.**

**Comment by OreCruncher** [comment]
Is it 100% pure Forge?  What is this Baritone stuff?  Also, from the log:

```
[12:41:19] [main/INFO] [mixin]: SpongePowered MIXIN Subsystem Version=0.7.11 Source=file:/C:/Users/Admin/AppData/Roaming/.minecraft/mods/baritone-standalone-forge-1.2.14.jar Service=LaunchWrapper Env=CLIENT
[12:41:19] [main/INFO] [mixin]: Compatibility level set to JAVA_8
[12:41:19] [main/WARN] [mixin]: Mixin config mixins.dsurround.json requires mixin subsystem version 0.8 but 0.7.11 was found. The mixin config will not be applied.
```
Baritone is causing the 0.7.11 version of Mixin to be loaded rather than 0.8.  Try installing this mod as well to see if it can force the order to cause the 0.8 version to load: https://www.curseforge.com/minecraft/mc-mods/mixin-0-7-0-8-compatibility.  (For clarity, the 0.8 version was the RELEASE version of Mixin.  0.7.11 is a BETA version.)

**Comment by Aaronek10** [comment]
Baritone is a kind of AI that I use to get to certain coordinates when I need to go AFK, or it can be used to mine certain resources/blocks
It's also kind of considered a cheat because it can do X-Ray stuff etc, but I use it more in a decent non-game breaking way (mainly in building stuff or as i mentioned, reach specified coordinates)

**Comment by Aaronek10** [comment]
Also i tried 0.7-0.8 compatibility but that does game won't launch, i mean, game launches but i can't see game window and cant do anything ;/

**Comment by OreCruncher** [comment]
Unfortunately there isn't much I can do.  You may want to create an issue for Baritone to get them to update their Mixin dependency.

**Comment by Aaronek10** [comment]
Yeah, i know, i downloaded a one version older DS and everything works fine, i'll try to fix this on my own, anyway thanks for your time and help!
